### PR TITLE
Show number of conversations

### PR DESCRIPTION
**Pull request by dagfait**
Show the total number of messages on the main page and show the number of conversations per company at the company stats.

### CODE REVIEW

**Comment by fatboypunk** [comment]
I think to merge this we need some tests around it.

**Comment by fatboypunk** [comment]
@GlennGeelen do you have a moment to look at this?

**Comment by GlennGeelen** [comment]
Yesterday something went wrong when importing some records, have you fixed that @dagfait ?

**Comment by dagfait** [comment]
The issue with importing the records is only present when trying to get un-tagged conversations, but that isn't in these commits yet. I think that should be a seperate pull request, as I view getting untagged conversations as a seperate issue.

File: lib/intercom_stats/repository/conversations.ex [hunk 1]
```diff
@@ -70,6 +70,29 @@ defmodule IntercomStats.Repository.Conversations do
...
+  defp map_number(list, name) do
+    %{
+      company_name: name,
+      number: get_number(list),
+    }
+  end
+
+  def get_number(conversations) do
+    conversations
+    |> count_messages()
```
Comment (GlennGeelen) [inline lib/intercom_stats/repository/conversations.ex]: no pipe needed here `count_message(conversations)`

File: lib/intercom_stats_web/controllers/company_controller.ex [hunk 2]
```diff
@@ -61,6 +61,9 @@ defmodule IntercomStatsWeb.CompanyController do
       %{tag: "gebruikersondersteuning"}
       |> Map.merge(filter)
       |> Conversations.conversation_averages_by_tag_and_company()
+    message_number =
+      filter
```
Comment (GlennGeelen) [inline lib/intercom_stats_web/controllers/company_controller.ex]: same here 
```
message_number = Conversations.conversation_number_by_company(filter)
```

**Pull request merged**